src/containers/PipelineRun/PipelineRun.js

```diff
@@ -165,7 +164,36 @@ export /* istanbul ignore next */ class PipelineRunContainer extends Component {
       );
     }
 
-    const { pipelineRun, taskRunNames } = this.loadPipelineRunData();
+    const {
+      error: pipelineRunError,
+      pipelineRun,
+      taskRunNames
+    } = this.loadPipelineRunData();
+
+    const {
+      lastTransitionTime,
+      reason: pipelineRunReason,
+      status: pipelineRunStatus
+    } = getStatus(pipelineRun);
+
+    if (pipelineRunError) {
+      return (
+        <>
+          <RunHeader
+            lastTransitionTime={lastTransitionTime}
+            loading={loading}
+            runName={pipelineRunName}
+            reason="Error"
+            status={pipelineRunStatus}
+          />
+          <InlineNotification
+            kind="error"
+            title="Unable to retrieve PipelineRun logs"
```

Comment:
This title is a little misleading, the actual error described is that the PipelineRun couldn't be created.

Maybe we should go with something more generic like 'Unable to load PipelineRun details' as there are likely other scenarios that can result in the same state. We could also include `pipelineRunError.reason` for additional info.

Reply:
Sounds good I will make that change 👍